src/test/scala/uk/gov/nationalarchives/tdr/api/db/repository/FileRepositorySpec.scala

```diff
@@ -229,6 +229,30 @@ class FileRepositorySpec extends TestContainerUtils with ScalaFutures with Match
       files.size shouldBe 1
   }
 
+  "getFiles" should "return files and file metadata where 'selectedFileIds' filter applied" in withContainers {
```

Comment:
Would reword the test slightly to: `... return files and file metadata for selected files where 'selectedFileIds' ...`

I would also add the directory to the selected files list to show it will bring back both types where the type filter is not applied.

Reply:
Sure, I'll add it too. Thanks